pkg/index/job/save/service/mock_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2019-2023 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package service
+
+import (
+	"context"
+
+	"github.com/vdaas/vald/internal/client/v1/client/discoverer"
+	"github.com/vdaas/vald/internal/net/grpc"
+)
+
+type mockDiscovererClient struct {
```

Comment:
It would be better to move this file to the `internal` directory.
There are mock files in the `internal` package.

Reply:
Thank you for suggestion! 🙏  It looks good to me!
I will create the mock file in `internal/test/mock`

Reply:
I have fixed 👉 https://github.com/vdaas/vald/pull/2227/commits/f2dc953c4ccb552287e173e23b15753d531cb3e3